numpyro/primitives.py

```diff
@@ -84,6 +84,10 @@ def sample(name, fn, obs=None, rng_key=None, sample_shape=()):
     :param numpy.ndarray obs: observed value
     :param jax.random.PRNGKey rng_key: an optional random key for `fn`.
     :param sample_shape: Shape of samples to be drawn.
+    :param dict infer: an optional dictionary containing additional information
+        for inference algorithms. For example, if `fn` is a discrete distribution,
+        setting `infer={'enumerate': 'parallel'}` will tell MCMC marginalize
```

Comment:
nit: `to tell`.

Reply:
We can fix this subsequently. I'm merging this so that your other PR is unblocked.